### ISSUE TITLE
Drush makefile error on Drupal.org

### ISSUE THREAD

**Issue by sheldonrampton** [civic-commons/opencivic#58]
Drupal.org's packaging script is refusing to accept the olfp module in this distro because it is built using a git repository that is not whitelisted:

http://drupal.org/node/1913594

**Comment by sheldonrampton** [comment]
I've filed a couple of tickets on Drupal.org to get this resolved:
- http://drupal.org/node/1913594
- http://drupal.org/node/1913686

Based on what Tim Plunkett told me on Drupal's IRC chat, it should take about three days for the repository to get whitelisted.

**Comment by sheldonrampton** [comment]
Aaron, I'm assigning this to you for code review. I did a whole bunch of work in this ticket, so I don't know how seriously you'll want to look through all of the code commits. I've removed the library dependency that was causing the packaging script to reject it on Drupal.org and have fixed some other bugginess in the distro. I should be able to get the packaging working now on d.o., but we're talking about renaming the project to "Open Civic" instead of "Apps Catalog," so I want to wait until the naming decision is made before trying again to publish it as a d.o. project, so I haven't actually tried it there yet.
